specs/binding-http.spec/src/main/resources/META-INF/zilla/http.idl

```diff
@@ -49,4 +49,77 @@ scope http
             HttpHeader[] headers;
         }
     }
+
+    scope event
+    {
+        enum HttpEventType (uint8)
+        {
+            CORS_PREFLIGHT_REQUEST_NOT_ALLOWED (1),
+            CORS_REQUEST_NOT_ALLOWED (2),
+            GUARD_AUTHORIZATION_FAILURE (3),
+            GUARD_AUTHORIZATION_SUCCESS (4),
+            ROUTE_NOT_FOUND (5),
+            ROUTE_FOUND (6)
+        }
+
+        enum Level
+        {
+            INFO,
+            WARNING,
+            ERROR
+        }
```

Comment:
The level can be determined by the caller, then this is just structural type information.

Reply:
The `Level` is now a field in the `HttpEventCommon` struct, its value is set in the methods of the `HttpEventContext` class. Does it look good? 🤔